addons/udes_stock/models/product_category.py

```diff
@@ -7,3 +7,100 @@ class ProductCategory(models.Model):
     active = fields.Boolean(
         default=True, string="Active", help="Display in list views or searches."
     )
+    u_suggest_locations = fields.Boolean(
+        string="Enable Suggest Locations",
+        default=False,
+        help="Computed suggest locations setting to use for this category.",
+        compute="_compute_suggest_locations",
+        store=True,
+    )
+    u_category_suggest_locations = fields.Boolean(
+        string="Enable Category Suggest Locations",
+        default=False,
+        help="When enabled a tab will be visible to suggest locations will be used for categories.",
+    )
+    u_suggest_location_ids = fields.Many2many(
+        "stock.location",
+        "suggested_locations_rel",
+        compute="_compute_suggest_location_ids",
+        store=True,
+        column1="category_id",
+        column2="location_id",
+        string="Suggest Locations",
+    )
+    u_category_suggest_location_ids = fields.Many2many(
+        "stock.location",
+        "product_category_suggested_locations_rel",
+        column1="category_id",
+        column2="location_id",
+        string="Category Suggest Locations",
+    )
```

Comment:
It might be helpful to have a comment that explains the difference between these two similar many to many relationships.

Reply:
Will try, one is computed and propogates into the children, and the other is a value user sets in the category

Reply:
I think I'm happy, @adamrpatrick feel free to resolve if you are too.